exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -14,156 +14,131 @@
  * limitations under the License.
  */
 
-use super::{Config, EjbConfig, JvmConfig, ServiceConfig};
+use super::{Config, PrivateConfig, PublicConfig};
 use exonum::helpers::fabric::keys;
```

Comment:
@alekseysidorov I remember soneone saying it would be better to get rid of this module and rewrite the whole thing. Couldn't agree more.

Reply:
It was @alekseysidorov and I totally agree with him.